README.md

```diff
@@ -181,6 +181,41 @@ pipeline {
 The shell command above will convert the PFX file to a pem key file (also containing the cert), note that Azure Key Vault removes the password
 on the pfx when you import it, if you're importing it back into Azure somewhere else you may need to convert it to pem and convert back to a pfx with a password.
 
+### Azure Key Vault Credentials Provider (Beta)
+
+This plugin also enables the retrieval of Secrets directly from Azure Key Vault. After the configuration is set up, secrets from the key vault can 
+be viewed in the credentials page like this:
+
+![provider](./docs/image/provider.png)
+
+*Note* These credentials are read-only and metadata caching(5 minutes) which means newly created secrets may not be here immediately.
```

Comment:
I'm thinking of two enhancements for this:
1. a button to just refresh?
2. an option to disable cache, some users may not want this.

thoughts?
can be follow-up issues for it as well

Reply:
Yes, cache can always introduce lots of related stuff. Should be labeled as enhancement issues.

Reply:
Done